### PR TITLE
BIOS steps: set power failure state.

### PR DESCRIPTION
**Pull request by thromer**
Also explicitly add the step to save BIOS settings.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @thromer, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19896).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_includes/installation/operating_system.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ To boot Home Assistant OS the BIOS needs to have UEFI boot mode enabled and Secu
 1. Disable Secure Boot
 ![Disable Secure Boot mode](/images/installation/intel-nuc-disable-secure-boot.jpg)
 
+1. Change Power:Secondary Power Settings:After Power Failure to "Power On" or "Last State".
+
+1. Press F10 to save BIOS settings.
```
Comment (frenck) [inline source/_includes/installation/operating_system.md:78]: I'm not sure why this is required? This is a personal choice, right?

Comment (thromer) [reply source/_includes/installation/operating_system.md:78]: Not required, but the common case is that you would want the system to come back up after a power failure. How about prefacing line 76 with "Optional: " ?

Comment (frenck) [reply source/_includes/installation/operating_system.md:78]: I don't think we should add this; as it is a side-thing. We also don't talk about using a UPS. This is not directly related to being able to install the operating system.

Nevertheless, thanks for willing to contribute 👍

**Pull request closed without merging**